tracer/build/_build/Build.Steps.cs

```diff
@@ -124,6 +124,11 @@ partial class Build
         Solution.GetProject(Projects.ToolIntegrationTests)
     };
 
+    Project[] DebuggerIntegrationTests => new[]
+    {
+        Solution.GetProject(Projects.DebuggerIntegrationTests)
+    };
+
```

Comment:
Will there ever be more than one debugger integration test? If not, I assume we can just inline this?

Reply:
+1, I don't think we'll ever have more than one.

Reply:
Fixed at 59340936a420da68670139fcd116e6b479f1c1d0